Realm/ObjectServerTests/RLMObjectServerTests.mm

```diff
@@ -388,7 +388,73 @@ - (void)testOtherUserChangePassword {
     }
 }
 
-// FIXME: write a 'testUserIsAdminFlagProperlySet' test once we can properly create admin users
+/// A sync admin user should be able to retrieve information about other users.
+- (void)testRetrieveUserInfo {
+    NSString *username1 = @"meela@realm.example.org";
+    NSString *username2 = @"jyaku@realm.example.org";
+    NSString *pw = @"p";
+    NSURL *server = [RLMObjectServerTests authServerURL];
+
+    // Create a non-admin user.
+    RLMSyncCredentials *c1 = [RLMSyncCredentials credentialsWithUsername:username1 password:pw register:YES];
+    RLMSyncUser *user1 = [self logInUserForCredentials:c1 server:server];
```

Comment:
Can we use more descriptive variable names? `user1` -> `nonAdminUser`, `user2` -> `adminUser1`, `testUser` -> `adminUser2`.